From 4f6533d3a44d96599cbd62ebf5309b14acdd2bc3 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Mon, 13 May 2019 13:16:35 +0200
Subject: [PATCH] arm: dt: fsl-imx8qxp-tqma8xqp-mbpa8xx: mark SD as non
 removable

CD is not supported, signal held to wrong logic level
while at it, disable also write protect


Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts b/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts
index 67ab66c242..365c9a33ed 100644
--- a/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts
+++ b/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts
@@ -197,8 +197,9 @@
 	pinctrl-1 = <&pinctrl_usdhc2_100mhz>, <&pinctrl_usdhc2_gpio>;
 	pinctrl-2 = <&pinctrl_usdhc2_200mhz>, <&pinctrl_usdhc2_gpio>;
 	bus-width = <4>;
-	cd-gpios = <&gpio4 22 GPIO_ACTIVE_LOW>;
-	wp-gpios = <&gpio4 21 GPIO_ACTIVE_HIGH>;
+	/* cd-gpios = <&gpio4 22 GPIO_ACTIVE_LOW>; */
+	non_removable;
+	/* wp-gpios = <&gpio4 21 GPIO_ACTIVE_HIGH>; */
 	vmmc-supply = <&reg_mbpa8xx_3v3>;
 	status = "okay";
 };
